easybuild/easyconfigs/c/CondaCreate/condacreate-1.0.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_CondaCreate'
+
+name = "condacreate-module"
```

Comment:
@jerowe I think the name used here should better reflect what is actually being installed...

This should probably be named `cpanminus` instead?

Does `conda` support providing a specific version in the requirements? Since as it is now, you have *no* control over which `cpanminus` version gets installed (and hence the `version` field can't be filled in meaningfully)...